pairing_api/examples/pairing.rs

```diff
@@ -0,0 +1,214 @@
+        match response {
+            Payload::Request(request) => match request.params {
+                Params::PairingDelete(_) => {
+                    // send a success response back to wc.
+                    let delete_request = ResponseParamsSuccess::PairingDelete(true);
+                    pairing_client
+                        .publish_response(&topic, delete_request, request.id, &client)
+                        .await
+                        .unwrap();
+                    // send a request to delete pairing from store.
+                    pairing_client.delete(&topic, &client).await.unwrap();
+                }
+                Params::PairingExtend(data) => {
+                    let extend_request = ResponseParamsSuccess::PairingExtend(true);
+                    // send a success response back to wc.
+                    pairing_client
+                        .publish_response(&topic, extend_request, request.id, &client)
+                        .await
+                        .unwrap();
+                    // send a request to update pairing expiry in store.
+                    pairing_client.update_expiry(&topic, data.expiry).await;
+                }
+                Params::PairingPing(_) => {
+                    let ping_request = ResponseParamsSuccess::PairingPing(true);
+                    // send a success response back to wc.
+                    pairing_client
+                        .publish_response(&topic, ping_request, request.id, &client)
+                        .await
+                        .unwrap();
+                }
+                _ => unimplemented!(),
+            },
+            Payload::Response(value) => {
+                println!("Response: {value:?}");
+            }
+        }
+    }
+}
+
+async fn pair_from_uri(pairing_client: &PairingClient, client: &Client) -> Topic {
+    let topic = pairing_client
+        .pair(
+            "wc:
+    b99c41b1219a6c3131f2960e64cc015900b6880b49470e43bf14e9e520bd922d@2?
+    expiryTimestamp=1725467415&relay-protocol=irn&
+    symKey=4a7cccd69a33ac0a3debfbee49e8ff0e65edbdc2031ba600e37880f73eb5b638",
+            true,
+        )
+        .await
+        .unwrap();
+    client.subscribe(topic.clone()).await.unwrap();
+    topic
+}
+
+// async fn create_pairing(pairing_client: &PairingClient) -> Topic {
+//     let metadata = Metadata {
+//         description: "A decentralized application that enables secure
+//         communication and transactions."
+//             .to_string(),
+//         url: "https://127.0.0.1:3000".to_string(),
+//         icons: vec![
+//             "https://example-dapp.com/icon1.png".to_string(),
+//             "https://example-dapp.com/icon2.png".to_string(),
+//         ],
+//         name: "Example DApp".to_string(),
+//     };
+
+//     let (topic, uri) = pairing_client.try_create(metadata,
+// None).await.unwrap();     println!("pairing_uri: {uri}");
+
+//     topic
+// }
```

Comment:
Do you still need it?

Reply:
yes, tester can use it for testing pairing creation.

Reply:
Could please you uncomment this code block, so it can track actual code changes, right now it has compile error
![image](https://github.com/user-attachments/assets/c3bb4499-d988-425a-8da9-4b1cf6ea7a40)

Provide doc comment description about for what this function can be used and mark and please add allow(dead_code) annotation then if its not currently used

Reply:
Resolved. 
Non blocking note please fix a typo in doc com. I suppose, its `create` a connection
https://github.com/KomodoPlatform/WalletConnectRust/blob/3ea808b981a7db0cb4c805bd156be0d48546a4fc/pairing_api/examples/pairing.rs#L197